source/data-process/picking-phase.rst

```diff
@@ -61,7 +61,8 @@ SAC 全局变量 ``SAC_PPK_USE_CROSSHAIRS`` 可以控制 ppk 模式下鼠标在
 窗口内的形态。若其值为 ``0``\ ，则鼠标会以十字线的形式出现，即
 “:math:`+`\ ”；当其值为 ``1`` 时，会在十字线的基础上加上水平线
 和垂直线。通常建议设置其值为 ``1``\ ，使得拾取到时时更精确。该全局
-变量的设置方式参考 :doc:`/install/linux-install` 一节。
+变量的设置方式参考 :doc:`/install/linux-binary` 或 :doc:`/install/linux-source`
```

Comment:
Same for this page as your suggestion in https://github.com/seisman/SAC_Docs_zh/pull/220/files#r642081750?

Reply:
Re-read it again. Maybe not a good idea to link to `install/index`

Reply:
?

Reply:
Let's link to the `install/index` doc now.

Reply:
done in 7f4b62c.